### DISCUSSION TITLE
Submit button not working

### DISCUSSION THREAD

**Discussion by NathanKarthik1996** [chaos-mesh/chaos-mesh#3826] [Q&A]
## Bug Report

**What version of Kubernetes are you using?**

Client Version: version.Info{Major:"1", Minor:"21", GitVersion:"v1.21.2", GitCommit:"092fbfbf53427de67cac1e9fa54aaa09a28371d7", GitTreeState:"clean", BuildDate:"2021-06-16T12:59:11Z", GoVersion:"go1.16.5", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"21+", GitVersion:"v1.21.14-eks-fb459a0", GitCommit:"b07006b2e59857b13fe5057a956e86225f0e82b7", GitTreeState:"clean", BuildDate:"2022-10-24T20:32:54Z", GoVersion:"go1.16.15", Compiler:"gc", Platform:"linux/amd64"}

**What version of Chaos Mesh are you using?**

chaos-mesh/chaos-mesh	2.5.0        	2.5.0      	Chaos Mesh is a cloud-native Chaos Engineering ...

**What did you do? / Minimal Reproducible Example**

I followed the production installation and used the docker method to install, I do see the dashboard but when I select my experiment and click the final submit button nothing seems to happen. I have tried uninstalling and installing chaos-mesh button seems to be persistent.
**What did you expect to see?**
Expected an experiment to be created
**What did you see instead?**
The same page with no action
![Screen Shot 2022-11-28 at 11 21 50 AM](https://user-images.githubusercontent.com/29498224/204329143-23913ca6-2ebf-4077-9788-2ffec3548f4d.png)

**Output of chaosctl**

**Comment by NathanKarthik1996** [comment]
Moreover, I have tried using the yaml method if launching the attacks. I have been having issues with the attacks. 

Failed to apply chaos: cannot find daemonIP on node ip-10-104-71-119.ec2.internal in related Endpoints {{Endpoints v1} {chaos-daemon  chaos-mesh  3aa2582f-5be8-4f64-8609-a09c6a7a7ac9 21115086 0 2022-11-28 19:28:35 +0000 UTC <nil> <nil> map[app.kubernetes.io/component:chaos-daemon app.kubernetes.io/instance:chaos-mesh app.kubernetes.io/managed-by:Helm app.kubernetes.io/name:chaos-mesh app.kubernetes.io/part-of:chaos-mesh app.kubernetes.io/version:2.5.0 helm.sh/chart:chaos-mesh-2.5.0 service.kubernetes.io/headless:] map[endpoints.kubernetes.io/last-change-trigger-time:2022-11-28T19:28:37Z] [] []  [{kube-controller-manager Update v1 2022-11-28 19:28:36 +0000 UTC FieldsV1 {"f:metadata":{"f:annotations":{".":{},"f:endpoints.kubernetes.io/last-change-trigger-time":{}},"f:labels":{".":{},"f:app.kubernetes.io/component":{},"f:app.kubernetes.io/instance":{},"f:app.kubernetes.io/managed-by":{},"f:app.kubernetes.io/name":{},"f:app.kubernetes.io/part-of":{},"f:app.kubernetes.io/version":{},"f:helm.sh/chart":{},"f:service.kubernetes.io/headless":{}}},"f:subsets":{}} }]} [{[{10.104.64.217  0xc0021993a0 &ObjectReference{Kind:Pod,Namespace:chaos-mesh,Name:chaos-daemon-545nh,UID:83e74aa5-7315-436a-b1be-1a01a463db96,APIVersion:,ResourceVersion:21115083,FieldPath:,}} {10.104.73.135  0xc0021993b0 &ObjectReference{Kind:Pod,Namespace:chaos-mesh,Name:chaos-daemon-tln24,UID:c1540e73-889a-4414-852e-5fca8de6c7f2,APIVersion:,ResourceVersion:21115042,FieldPath:,}} {10.104.83.224  0xc0021993c0 &ObjectReference{Kind:Pod,Namespace:chaos-mesh,Name:chaos-daemon-76z95,UID:f89d09ea-42be-40b3-9143-3f6757e74ed0,APIVersion:,ResourceVersion:21115047,FieldPath:,}} {10.104.85.251  0xc0021993d0 &ObjectReference{Kind:Pod,Namespace:chaos-mesh,Name:chaos-daemon-nw5w2,UID:dbfdc379-88c7-4ce8-baf8-82fe2c02d900,APIVersion:,ResourceVersion:21115057,FieldPath:,}}] [] [{grpc 31767 TCP <nil>} {http 31766 TCP <nil>}]}]}

**Reply by fewdan** [reply]
It seems to be a problem with chaos-daemon, can you provide the results of the following commands?
`kubectl describe ds -n chaos-testing chaos-daemon`

**Reply by NathanKarthik1996** [reply]
Name:           chaos-daemon
Selector:       app.kubernetes.io/component=chaos-daemon,app.kubernetes.io/instance=chaos-mesh,app.kubernetes.io/name=chaos-mesh
Node-Selector:  <none>
Labels:         app.kubernetes.io/component=chaos-daemon
                app.kubernetes.io/instance=chaos-mesh
                app.kubernetes.io/managed-by=Helm
                app.kubernetes.io/name=chaos-mesh
                app.kubernetes.io/part-of=chaos-mesh
                app.kubernetes.io/version=2.5.0
                helm.sh/chart=chaos-mesh-2.5.0
Annotations:    deprecated.daemonset.template.generation: 1
                meta.helm.sh/release-name: chaos-mesh
                meta.helm.sh/release-namespace: chaos-mesh
Desired Number of Nodes Scheduled: 6
Current Number of Nodes Scheduled: 6
Number of Nodes Scheduled with Up-to-date Pods: 6
Number of Nodes Scheduled with Available Pods: 6
Number of Nodes Misscheduled: 0
Pods Status:  6 Running / 0 Waiting / 0 Succeeded / 0 Failed
Pod Template:
  Labels:           app.kubernetes.io/component=chaos-daemon
                    app.kubernetes.io/instance=chaos-mesh
                    app.kubernetes.io/managed-by=Helm
                    app.kubernetes.io/name=chaos-mesh
                    app.kubernetes.io/part-of=chaos-mesh
                    app.kubernetes.io/version=2.5.0
                    helm.sh/chart=chaos-mesh-2.5.0
  Annotations:      rollme: 2cJBi
  Service Account:  chaos-daemon
  Containers:
   chaos-daemon:
    Image:       ghcr.io/chaos-mesh/chaos-daemon:v2.5.0
    Ports:       31767/TCP, 31766/TCP
    Host Ports:  0/TCP, 0/TCP
    Command:
      /usr/local/bin/chaos-daemon
      --runtime
      docker
      --http-port
      31766
      --grpc-port
      31767
      --pprof
      --ca
      /etc/chaos-daemon/cert/ca.crt
      --cert
      /etc/chaos-daemon/cert/tls.crt
      --key
      /etc/chaos-daemon/cert/tls.key
      --runtime-socket-path
      /host-run/docker.sock
    Environment:
      TZ:  UTC
    Mounts:
      /etc/chaos-daemon/cert from chaos-daemon-cert (ro)
      /host-run from socket-path (rw)
      /host-sys from sys-path (rw)
      /lib/modules from lib-modules (rw)
  Volumes:
   socket-path:
    Type:          HostPath (bare host directory volume)
    Path:          /var/run
    HostPathType:  
   sys-path:
    Type:          HostPath (bare host directory volume)
    Path:          /sys
    HostPathType:  
   lib-modules:
    Type:          HostPath (bare host directory volume)
    Path:          /lib/modules
    HostPathType:  
   chaos-daemon-cert:
    Type:        Secret (a volume populated by a Secret)
    SecretName:  chaos-mesh-daemon-certs
    Optional:    false
Events:
  Type    Reason            Age   From                  Message
  ----    ------            ----  ----                  -------
  Normal  SuccessfulCreate  17m   daemonset-controller  Created pod: chaos-daemon-b6fqm
  Normal  SuccessfulCreate  17m   daemonset-controller  Created pod: chaos-daemon-lvbg6
  Normal  SuccessfulCreate  17m   daemonset-controller  Created pod: chaos-daemon-jmwdl
  Normal  SuccessfulCreate  17m   daemonset-controller  Created pod: chaos-daemon-b2fsc
  Normal  SuccessfulCreate  17m   daemonset-controller  Created pod: chaos-daemon-gqk7c
  Normal  SuccessfulCreate  17m   daemonset-controller  Created pod: chaos-daemon-f5cgv

**Reply by fewdan** [reply]
It seems that chaos-daemon is normal, but your chaos-dashboard still cannot inject chaos?

**Comment by bumarcell** [comment]
Any update here? I'm having a similar issue where the chaos experiment is created but then with the same error `cannot find daemonIP on node <nodeName> in related Endpoints {{Endpoints v1}...`
The daemonset seems to be correct and what I find interesting is that the node from the error isn't included in the daemonset.

**Reply by bumarcell** [reply]
I guess my problem was that I was selecting some wrong pods  in the inject info target, which belonged to some daemonset and were running on nodes without a chaos-daemon pod.
